build-haero.sh

```diff
@@ -73,6 +73,7 @@ fi
 echo "Cloning Haero repository into $(pwd)/.haero..."
 git clone git@github.com:eagles-project/haero.git .haero || exit
 cd .haero || exit
+git checkout -t origin/kylepres/surface.hpp 
```

Comment:
Great. We should delete this line after Haero gets merged (and add `[rebuild-haero]` to that commit)